### ISSUE TITLE
Gender Field (YvonneScrivener)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#294]
Issue reported by **YvonneScrivener** at 2015-07-20 06:19:01 UTC
Time: 2015-07-20T06:12:54+00:00
Session ID: 8b0245ef0212362b762949ac39c87a8a
Problem Page URL: [/freereg1_csv_entries/5501f67ae93790decc0443e2](/freereg1_csv_entries/5501f67ae93790decc0443e2)
Previous Page URL: []()
Reported Issue:
Using the TWYS principle, I have been recording the gender in baptism files with a hyphen if the register doesn't record son or daughter. 

However, I have been alerted to the fact that at least in one file, a question mark is displaying in the Search Results instead of the hyphen. (File SOMSMOBA1.CSV.2481 as per Data Error Report that came to me today).

How can I fix this so that the hyphen is displaying correctly for researchers?

The files were checked / transcribed using Excel, not WinReg, and uploaded via FreeREG1.

Thank you,

Yvonne Scrivener (Somerset - Frome Area Assistant)

![Screenshot](http://freereg2.freereg.org.uk)

**Comment by PatReynolds** [comment]
Dear Yvonne,

Thanks for this - we'll look into this.

With best wishes,

Pat

Pat Reynolds,
Executive Director, Free UK Genealogy

**Comment by Captainkirkdawson** [comment]
This is a very interesting subject that requires careful consideration as to how we proceed. Firstly sex is not a field in a register. It is a derived field based on the information available to the transcriber in the register. I believe WinReg has a pick list for the field. No sure what all of its options are. In the early transcription the field was a string field. As a result transcribers entering data into the field entered a variety of strings.
An analysis of the contents of FR1 identified a great variety of values these were classed as providing 
VALID_MALE_SEX = ["M","M." ,"SON","MALE","MM","SON OF"]
VALID_FEMALE_SEX = ["F","FF","FFF","FM","F.","FEMALE","DAUGHTER","WIFE","DAUGHTER OF","DAU", "DAU OF"]
UNCERTAIN_MALE_SEX = ["M?","SON?","[M]" ,"MF"]
UNCERTAIN_FEMALE_SEX = ["F?", "DAU?"]
UNCERTAIN_SEX = ["?", "-", "*","_","??"]
FR2 displays M, M?, F, F? and ? for these variations in values respectively.

This provides a consistent information display of the field which as noted above is a derived field rather than a transcribed field.

**Comment by PatReynolds** [comment]
Thanks, Kirk.

This needs to be two fields, I think.

One needs to be a type-what-you-see transcription.  The second needs to be
metadata interpretation.

Unfortunately, I can't reach the Internet at the moment to check what the
standard terms used by the GRO are, but it might be good to adopt those. I
would prefer to avoid abbreviations.

Best wishes,

Pat

**Comment by PatReynolds** [comment]
But abbreviations take less screen space ...

Cheers.

Pat

**Comment by Captainkirkdawson** [comment]
Remember these are Parish Registers for which there is no standard; until 1837 they are what the cleric of the day decided to put in the register

**Comment by benwbrum** [comment]
I'd also ask what we'll do with the derived field.  I have no objection to
derived fields--we do a lot with them under the hood in the search engine
for expanding abbreviations and sorting dates--but we generally used the
derived field internally and present the verbatim entry to the researcher.

Are we taking about allowing the researcher to search based on the sex of
the individual?  If so, it's doable, but requires (relatively
straightforward) work on the search engine and implementation of
standardized derived fields.  If not, I'm not sure what the benefit is.

Ben

**Comment by PatReynolds** [comment]
For transcription, it doesn't matter, as no standardisation is needed.

For metadata, if the GRO is still just using m and f (or things that mean
the same) then we don't need to change.  But if their records now include
other identifications which will be coming into BMD and other records we
should plan to add those to the drop-down list towards the end of the
project.

Best wishes,

Pat

**Comment by Captainkirkdawson** [comment]
I have moved this story back into incoming as I no longer understand what we want done under this story.

**Comment by PatReynolds** [comment]
(a) clarify how to record gender when words such as 'daughter of' are missing or cannot be interpreted (dash, question mark, or something else?
(b) ensure consistency in recording
(c) ensure character displayed is the character recorded

**Comment by edickens** [comment]
```
I assume this the Baptism record.  We will need Gender fields for
marriages soon to allow for same sex marriages.

E

On 20/10/2015 15:26, PatReynolds wrote:


  (a) clarify how to record gender when words such as 'daughter
    of' are missing or cannot be interpreted (dash, question mark,
    or something else?
    (b) ensure consistency in recording
    (c) ensure character displayed is the character recorded
  —
    Reply to this email directly or view
      it on GitHub.
```

**Comment by Captainkirkdawson** [comment]
Included as part of #809 flexible_csv
